content/_data/changelogs/lts.yml

```
@@ -12626,6 +12626,324 @@
+      Upgrade to Winstone 8.7 and Jetty 12.0.19.
+
+  lts_changes: # compared to lts_predecessor 2.492.3 (selected by personal review)
+
+  - type: major rfe
+    category: rfe
+    pull: 10135
+    issue: 75100
+    authors:
+      - timja
+    pr_title: Remove YUI
+    message: |-
+      Remove the Yahoo! User Interface library.
+  - type: major rfe
+    category: rfe
+    pull: 10147
+    authors:
+      - timja
+      - janfaracik
+    pr_title: "Add experimental 'Details' widget for builds"
+    message: |-
+      Add experimental <code>Details</code> widget for builds.
+  - type: rfe
+    category: rfe
+    pull: 10141
+    issue: 74832
+    authors:
+      - mawinter69
+    pr_title: "[JENKINS-74832] disable copybutton in insecure context"
+    message: |-
+      Disable the copy button in insecure contexts.
+  - type: rfe
+    category: rfe
+    pull: 10139
+    authors:
+      - janfaracik
+      - timja
+    pr_title: Update the 'Copy' button animation
+    message: |-
+      Update the <strong>Copy</strong> button animation.
+  - type: rfe
+    category: rfe
+    pull: 10122
+    authors:
+      - basil
+    pr_title: Remove Commons Discovery
+    message: |-
+      Remove Commons Discovery from the Jenkins WAR.
+  - type: rfe
+    category: rfe
+    pull: 10115
+    authors:
+      - janfaracik
+      - timja
+    pr_title: Display Console Output on the build page (behind an experimental
+      flag)
+    message: |-
+      Add the option to display the <strong>Console Output</strong> on the build page behind an experimental flag.
+  - type: rfe
+    category: rfe
+    pull: 10119
+    authors:
+      - janfaracik
+      - krisstern
+    pr_title: Wrap app bars on smaller screens
+    message: |-
+      Wrap app bars on smaller screens.
+  - type: rfe
+    category: rfe
+    pull: 8523
+    authors:
+      - jtnord
+      - MarkEWaite
+      - basil
+    pr_title: "Remove jCIFS and j-Interop"
+    message: |-
+      Remove the ability to install as a Windows service from a running Jenkins install.
+      Users wanting this functionality should instead <a href="/download/">download and install</a> the MSI package.
+  - type: rfe
+    category: rfe
+    pull: 10143
+    authors:
+      - janfaracik
+    pr_title: Refresh the interface of Jenkins CLI
+    message: |-
+      Refresh the interface of Jenkins CLI page.
+  - type: rfe
+    category: rfe
+    pull: 10078
+    authors:
+      - janfaracik
+      - timja
+      - krisstern
+    pr_title: Use `oklch` for the Jenkins UI
+    message: |-
+      Define Jenkins UI colors in a more perceptually uniform way using <code>OKLCH</code>.
+  - type: rfe
+    category: rfe
+    pull: 10180
+    authors:
+      - janfaracik
+    pr_title: Improve the appearance of user avatars in Jenkins
+    message: |-
+      Improve the appearance of user avatars in Jenkins.
+      Custom avatars are also avaialble through the <a href="https://plugins.jenkins.io/avatar/"Avatar plugin</a>. 
```

Spelling fix and a markup correction for the link.
```suggestion
      Custom avatars are also available through the <a href="https://plugins.jenkins.io/avatar/">Avatar plugin</a>. 
```